benchmark/pyproject.toml

```diff
@@ -9,25 +9,26 @@ packages = [{ include = "agbenchmark" }]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-# helicone = "^1.0.9"  # incompatible with openai@^1.0.0
-agent-protocol-client = "^1.1.0"
+pydantic = "^2.7.2"
+pydantic-settings = "^2.3.4"
 click = "^8.1.3"
 click-default-group = "^1.2.4"
 colorama = "^0.4.6"
 fastapi = "^0.109.1"
 gitpython = "^3.1.32"
 httpx = "^0.24.0"
 matplotlib = "^3.7.2"
+multidict = "^6.0.5"
```

Comment:
`multidict 6.0.4` fails to install and is a dependency of some package, that has been a reoccurring issue.